cmd/pipe-main.go

```diff
@@ -183,6 +189,9 @@ func mainPipe(ctx *cli.Context) error {
 	// validate pipe input arguments.
 	checkPipeSyntax(ctx)
 
+	// globalQuiet is true for no window size to get. We just need --quiet here.
+	quiet := ctx.IsSet("quiet")
```

Comment:
We should add `--quiet` to command line options as well at Line#37. Also add help content about this flag.

Reply:
this flag is global flag @shtripat

Reply:
ah thats my bad. thanks